cypress/e2e/patient_spec/encounter-notes.cy.ts

```
@@ -0,0 +1,78 @@
+    // logging as nurse
+    cy.loginByApi("devnurse");
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC Trikaripur");
+    patientEncounter.navigateToEncounters().openFirstEncounterDetails().openNotesTab();
+  });
+
+  it("create a new note in the Notes tab", () => {
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .openNotesTab()
```

ESLint

[error] Insert `;`

(prettier/prettier)